.travis.yml

```diff
@@ -24,7 +24,10 @@ env:
   matrix:
     - RUN_LINT=true
     - RUN_FRONTEND_TESTS=true
-    - RUN_E2E_TESTS=true
+    - RUN_E2E_TESTS_FULL=true
```

Comment:
I don't see the need to run 'full' as well as the others -- it just seems like unnecessary duplication, and does not add new information. Let's remove it?

Also, I want to suggest pulling out explorationRating.js and publicationAndLibrary.js (actually, oops, this name has changed, I'm kind of surprised that everything ran without errors) into a separate "library" suite. The publicationAndLibrary.js test tends to fail, so I think it makes sense to isolate it a little more. Also, let's move historyTab.js and fallbacks.js into editor, since that's what those are testing.

OK, this is starting to turn into a full reorganization. Here's what I suggest:

editor:
 - editorAndPlayer, fallbacks, gadgetEditor, historyTab, stateEditor

extensions:
- interactions, richTextComponents

library:
- explorationRating, privileges, publicationAndLibrary

misc:
- embedding, preferences, userManagement

Does that sound good?